client/lib/checkout/validation.js

```diff
@@ -230,6 +231,19 @@ validators.validPostalCodeUS = {
 	},
 };
 
+validators.validPhone = {
+	isValid( phoneNumber, paymentDetails ) {
+		if ( ! phoneNumber ) {
+			return false;
+		}
+
+		return phone( phoneNumber, paymentDetails.country ).length > 0;
+	},
+	error: function() {
+		return i18n.translate( 'That phone number does not appear to be valid' );
```

Comment:
The error function is passed in a parameter with the translated label text. I think its done for consistency reasons but I'm on the fence how useful I think it is translation wise. Might be worth using it to remain consistent even if the translated result is less than perfect.

Reply:
I used this specific string because it's already translated. Together with the change on the rules stacking, I think it's even better now.